web/templates/settings/settings_checkbox.hbs

```diff
@@ -1,11 +1,11 @@
 {{#unless (eq render_only false)}}
-<div class="input-group {{#if is_disabled}}control-label-disabled{{/if}} {{#if tooltip_message}}tippy-zulip-tooltip{{/if}}" {{#if tooltip_message}}data-tippy-content="{{tooltip_message}}"{{/if}}>
+<div class="input-group {{#if is_disabled}}control-label-disabled{{/if}}">
     <label class="checkbox">
         <input type="checkbox" class="{{setting_name}} inline-block setting-widget {{#unless skip_prop_element}}prop-element{{/unless}}" name="{{setting_name}}" data-setting-widget-type="boolean"
           id="{{#if prefix}}{{prefix}}{{/if}}{{setting_name}}" {{#if is_checked}}checked="checked"{{/if}} {{#if is_disabled}}disabled{{/if}} />
         <span class="rendered-checkbox"></span>
     </label>
-    <label for="{{#if prefix}}{{prefix}}{{/if}}{{setting_name}}" class="inline {{setting_name}}_label" id="{{#if prefix}}{{prefix}}{{/if}}{{setting_name}}_label">
+    <label for="{{#if prefix}}{{prefix}}{{/if}}{{setting_name}}" class="inline {{setting_name}}_label{{#if tooltip_message}} tippy-zulip-tooltip{{/if}}" id="{{#if prefix}}{{prefix}}{{/if}}{{setting_name}}_label"{{#if tooltip_message}} data-tippy-content="{{tooltip_message}}"{{/if}}>
```

Comment:
This change means that the tooltip is not shown when hovering over the checkbox and it is shown only when hovering over label. I think for other similar checkboxes, we add a containing div and set `width: fit-content` on those to not show tooltip when hovering over the empty space.